### DISCUSSION TITLE
I few ideas

### DISCUSSION THREAD

**Discussion by GeoDerp** [iamrasel/lunar-launcher#89] [Ideas]
Greetings,
while using the launcher for the first time some ideas came to mind that I think could be good quality of life improvements:
- option to message contact if you double click in quick access
- weather has an option to display hr'ly or day'ly weather forecast
- ability to disable apps in appdraw  
- ability to change the order of the todo list
- multiple rss feed support (if not implemented yet)

I may be able to help integrate these if I have time in the end of the year. These are just ideas, not need to implement any of them

**Comment by iamrasel** [comment]
Thanks @GeoDerp for sharing your ideas. My thoughts on them -
- this one could be done soon.
- can't say if i'll work to enhance the weather forecast thing in near future, but i'll keep this in my consideration.
- I guess you wanted to say hide by the word `disable`, i'll start working on this soon.
- how would you want to order them? a-z?
- nope, it hasn't been implemented yet.

**Reply by GeoDerp** [reply]
Amazing, Thank you!

> how would you want to order them? a-z?

Order wise, I guess i'm thinking being able to drag a todo task from bottom to top. *(This could already be implemented. If so, I haven't worked out how )*

Thanks for your response, appreciate it.

**Reply by GeoDerp** [reply]
This may be unnecessary, what do you think about the idea of adding an option to insert the image tags into the RSS feeds. The RSS image can be inserted under the RSS heading. Could possibly place an black and white filter on the images to keep to the minimalist theme.

This may require each RSS article to be in separate container, and may look clunky. So I understand if you don't agree with the idea,
May be something I can look into implementing as a pull request if I ever have the time.
